File: GitNexus/gitnexus/test/fixtures/sample-code/simple.swift
class UserManager {
    var users: [String] = []

    init() {
        users = []
    }

    func addUser(_ name: String) {
        users.append(name)
    }

    public func getCount() -> Int {
        return users.count
    }
}

func helperFunction() -> String {
    return "swift helper"
}

File: GitNexus/gitnexus/test/fixtures/sample-code/simple.cpp
#include <string>

class UserManager {
public:
    void addUser(const std::string& name) {
        users_.push_back(name);
    }

    int getCount() const {
        return static_cast<int>(users_.size());
    }

private:
    std::vector<std::string> users_;
};

int helperFunction(int x) {
    return x * 2;
}
